README.md

```diff
@@ -47,3 +47,23 @@ Decode a binary:
             97,42,106,106>>,
     bert:decode(Bin).
     > #{key => [{some, "tuple"}, {true, 42}]}
+
+## Documentation generation
```

Comment:
I've used a different style for section headers befire. Could you match the new documentation to that OR update the older ones?

Reply:
> I've used a different style for section headers befire. Could you match the new documentation to that OR update the older ones?

I can update the older ones.